### ISSUE TITLE
Incompatible with Thunderbird 11 (current beta)

### ISSUE THREAD

**Issue by alexmurray** [gnome-integration-team/thunderbird-gnome#5]
The title basically says it all - firstly it won't install (since maxversion is currently 9.0ish) and secondly it doesn't seem to handle the new tabs on top style of TB 11 very well - there are a few visual inconsistencies. Would be great to see an updated version.

**Comment by garrett** [comment]
I plan to work on this sometime within the next week. Sorry for the inconvenience.

**Comment by alexmurray** [comment]
No worries - don't feel bad, we all have different priorities - I'm just glad to have good designers such as yourself taking the time to make Gnome etc as great as it can be. Thanks.

**Comment by garrett** [comment]
I am working on this in the update-wip branch and have uploaded a snapshot in the downloads area for Thunderbird 11:
https://github.com/downloads/adwaita-firefox-team/adwaita-thunderbird/adwaita-thunderbird-11pre1.xpi

It's not quite done yet (hence the WIP in the branch name), but hopefully I can get it to a releasable state sometime in the next few days.

(On the other hand, what's there now is obviously better than not having the theme, so it might be good enough for a release, and then we update it more properly later? Not sure.)

**Comment by alexmurray** [comment]
Nice - yep this definitely looks better, thanks muchly for the speedy update.

**Comment by alexmurray** [comment]
I've noticed the latest Adwaita (for GNOME 3.4) includes a new scrollbar - any chance of this too? 
https://live.gnome.org/GnomeOS/Design/Whiteboards/Scrolling#Tentative_Design

**Comment by garrett** [comment]
Yep! The Firefox nightly theme has it; I plan on porting it to Thunderbird also, when I port the 3.4 changes.

I'm intending on both landing around the same time. (Basically, when Gecko/Firefox/Thunderbird 13 lands, as it changes the way tabs are specified anyway.)

**Comment by alexmurray** [comment]
Cool, thanks Garret sounds great.
